Packs/ProofpointIsolation/README.md

```diff
@@ -0,0 +1,16 @@
+### Proofpoint Isolation
+Proofpoint Isolation is a fully managed cloud-based solution that provides secure web browsing by isolating user sessions, protecting against threats like malware, phishing, and data breaches.
+
+### Proofpoint Isolation Event Collector:
```

Comment:
same as comment below
Better add a few sentences about what the product does, what are the use-cases